blocks/cards/cards.css

```diff
@@ -35,6 +35,12 @@
   aspect-ratio: 764 / 355;
 }
 
+@media (max-width: 768px) {
+  .card .card-thumbnail img {
+    aspect-ratio: auto;
```

Comment:
Without any aspect ratio (or height) defined the LHS score is bad, because it create CLS (Content Layout Shift).

The browser doesn't know how much space to leave for the images, so when the image loads, it pushes the other content around it. Additionally (I haven't tested), I think that if somebody uses an image with incorrect sizing, it will break the cards look.